### PR TITLE
style(components): [upload] dragger unlimited width

### LINKED ISSUE #7316: [Bug Report] [Style] [upload] 上传组件/拖拽上传 组件样式需优化
Snipaste_2022-04-22_01-07-32

Bug Type: **`Style`**

## Environment

- Vue Version: `3.2.33`
- Element Plus Version: `2.1.10`
- Browser / OS: `Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/100.0.4896.127 Safari/537.36 Edg/100.0.1185.44`
- Build Tool: `Vite`

### Related Component

- `el-upload`

### Reproduction Link

[Docs CN](https://element-plus.gitee.io/zh-CN/component/upload.html#%E6%8B%96%E6%8B%BD%E4%B8%8A%E4%BC%A0)

### Steps to reproduce

直接查看演示文档即可

### What is Expected?

样式类.el-upload-dragger不要限定容器具体的宽高，而是根据父容器的大小自动填充

### What is actually happening?

样式类.el-upload-dragger限定了容器的宽和高，width: 360px;height: 180px;

### Additional comments

看了下其他框架，例如ant-vue,naive-ui等都是将width设定为100%

### PR DESCRIPTION
**Pull request by josonho**
fix #7316 

I added a class to control the style of the drag

***

Please make sure these boxes are checked before submitting your PR, thank you!

- [x] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [x] Make sure you are merging your commits to `dev` branch.
- [x] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

**Comment by YunYouJun** [comment]
image
image

Your modification affects many other issues, please only modify the upload drag part, not the overall style.

**Comment by josonho** [comment]
> image
> 
> image
> 
> Your modification affects many other issues, please only modify the upload drag part, not the overall style.

please review，thanks.

**Comment by josonho** [comment]
modified, thanks.

File: packages/theme-chalk/src/upload.scss [hunk 1]
```diff
@@ -124,6 +127,7 @@
   }
 
   @include when(dragover) {
+    padding: 39px 9px;
```
Comment (YunYouJun) [inline packages/theme-chalk/src/upload.scss:130]: Can we set variables for `40px 10px`?  And minus 1px to get 39px instead of write 39px directly.

**Review by YunYouJun — CHANGES_REQUESTED** [review]

File: packages/theme-chalk/src/upload.scss (hunk 1 above)
Comment (josonho) [reply packages/theme-chalk/src/upload.scss]: of course

File: packages/theme-chalk/src/upload.scss [hunk 2]
```diff
@@ -84,7 +88,8 @@
 }
 
 @include b(upload-dragger) {
-  padding: 40px 10px;
+  padding: var(--el-upload-dragger-padding-horizontal)
```
Comment (YunYouJun) [inline packages/theme-chalk/src/upload.scss:91]: Please use `getCssVar` to generate css var. (So we can change namespace).

Just refer https://github.com/element-plus/element-plus/pull/7344/

**Review by YunYouJun — CHANGES_REQUESTED** [review]

**Review by YunYouJun — APPROVED** [review]

**Pull request merged**